src/components/ScanOrder.js

```diff
@@ -196,7 +196,7 @@ const ScanOrder = (props) => {
               <div className="form-group">
                 <label htmlFor="next_status">
                   Next Status:{" "}
-                  {STATUSES && order ? (
+                  {statuses && order.status.description ? (
                     <strong>
```

Comment:
What about for Cancelled? How does that display here?

Reply:
I don't want people to accidentally roll into it.  Was planning on giving it a separate button all it's own but am open to feedback.

Reply:
Ah I think I misunderstood what the code is doing here. No worries!